db/seed.js

```diff
@@ -1,71 +0,0 @@
-const mongoose = require("mongoose");
-const User = require("../models/user");
-const Review = require("../models/review");
-const Company = require("../models/company");
```

Comment:
Doesn't look like you're using this file, always good practice to remove what you don't need - it keeps the weight of the application down